src/py42/util.py

```diff
@@ -72,3 +72,12 @@ def format_dict(dict_, label=None):
     if label:
         return u"{} {}".format(label, indented_dict)
     return indented_dict
+
+
```

Comment:
So this is technically a public facing method that does not have documentation.  Do you mind also adding documentation for this method since it is now public facing?  It could be simple like """Returns attribute names on the given class""" And then have the normal `Args:` and `Returns:` parts.

Reply:
The docs will show up the `methoddocs/util` but you should not have to do anything except add the doc string for it to work.